03-Azure/01-03-Infrastructure/04_BCDR_Azure_Native/Infra/App1/.gitignore

```
@@ -0,0 +1,2 @@
+WebApp1
+.vs
```

wouldnt this have to be *.vs to work?
```suggestion
*.vs
```

---

reviewed - .vs fine